setup.py

```diff
@@ -17,6 +17,7 @@
     "Click>=7.0",
     "mistune>=2.0.0a4",
     "pyfiglet>=0.8.post1",
+    "Pygments>=2.10.0",
```

Comment:
`pygments` is the library that adds highlighting to the code. It has different styles/themes, but I'm leaving it on the default.